public/css/theme-kobo.css

```diff
@@ -2219,7 +2219,7 @@
   box-sizing: border-box; }
 
 html {
-  font-size: 62.5%;
+  /*font-size: 62.5%;*/
```

Comment:
Maybe just delete this line rather than comment it out?

Reply:
I made this temporary, if it is working perfectly everywhere then we can remove.